File: api_signup/get_apikey.py
# pip install requests
# python3 get_apikey.py
import requests, json
base_url = 'https://solve.shimul.me/api/'



# Fill details bellow before making request. Password are encrypted before saving it to database
# Remember no temp-mail. Temp-mail = ban permanently.
user_id=''
email=''
password = ''
name = ''


signup_data={}
signup_data['user_id']=user_id
signup_data['email']=email
signup_data['name']=name
signup_data['password']=password

r = requests.post(url = base_url+'account/signup', headers={'Content-Type': 'application/json'}, data = json.dumps(signup_data))


# print(r)
print(json.dumps(r.json()))

File: api_signup/get_apikey.js
const axios = require('axios');
const base_url = 'https://solve.shimul.me/api/'


// Fill details bellow before making request
// Remember no temp-mail. Temp-mail = ban permanently.
const user_id=''
const email=''
const password = ''
const name = ''


const result = axios({
  method: 'post',
  url: base_url+'account/signup',
  headers: {
  'Content-type': 'application/json'
  },
  data: data={"user_id": user_id, "email": email, "password": password, "name": name}
})
 .then((response) => {
    console.log(response.status);
    console.log(response.data)
})
.catch((error) => {
    console.log(error);
  });
